config/env/development.cfg

```diff
@@ -15,6 +15,7 @@ recipe = zc.recipe.egg:scripts
 eggs =
   manoseimas-lt
   nose
+  django-webpack-loader
```

Comment:
Best bet is to add this to base.cfg because production will need this as well - assets will need to be built on the production machine when deploying.